nncf/torch/pruning/operations.py

```diff
@@ -105,7 +106,7 @@ class PTOutputPruningOp(OutputPruningOp, PTPruner):
 @PT_PRUNING_OPERATOR_METATYPES.register('identity_mask_propagation')
 class PTIdentityMaskForwardPruningOp(IdentityMaskForwardPruningOp, PTPruner):
     subtypes = [HardTanhMetatype, TanhMetatype, RELUMetatype, PRELUMetatype, ELUMetatype, GELUMetatype, SigmoidMetatype,
-                SoftmaxMetatype, AvgPool2dMetatype, MaxPool2dMetatype, DropoutMetatype, SILUMetatype]
+                SoftmaxMetatype, AvgPool2dMetatype, MaxPool2dMetatype, DropoutMetatype, SILUMetatype, LeakyRELUMetatype]
```

Comment:
This doesn't guarantee that pruning operation would be added for the model with LeakyReLU in a particular customer case. Ideally, there should a test with pruning a very basic synthetic model, e.g. https://github.com/openvinotoolkit/nncf/blob/develop/tests/torch/test_compressed_graph.py#L728
or probably, there's already a similar test in the pruning folder.

Reply:
discussed internally. The proposed test would be redundant, because IdentityMask (based on ReLU) is properly tested.